app/views/registers/_fields.html.haml

```diff
@@ -0,0 +1,23 @@
+
+%table.registers-secondary-table
+  %caption.registers-secondary-table__caption
+    The
+    <!-- -->
+    %b #{@register.register_fields.count} fields
+    <!-- -->
```

Comment:
Can we move these comment blocks?

Reply:
Unfortunately not if we want spaces between the words - Haml is set up to strip out whitespace, even here when you've got nested elements.

Reply:
can we just use in-line HTML instead e.g. 
```
The <strong>#{@register.register_fields.count}</strong> fields in this register
```

Reply:
Sure - using comments was something that was used elsewhere in this repo, so I was trying to keep thing consistent. If I see the other uses of this I'll change them to inline as it is more readable.